README.md

```diff
@@ -1237,3 +1237,32 @@ DELETE /api/v1/users/mary
 ```
 HTTP/1.1 200 OK
 ```
+
+# Upgrades
+
+Initiates an upgrade to the provided version. Once this endpoint is called [Horticulturalist](https://github.com/medic/horticulturalist) will take over and complete the installation.
+
+### Permissions
+
+`_admin` (CouchDB Administrator)
+
+## Example
+
+```
+POST /api/v1/upgrade
+{
+  "build": {
+    "namespace": "medic",
+    "application": "medic",
+    "version": "3.0.0-beta.1"
+  }
+}
+```
+
```

Comment:
I'm not crazy about this API. I'd prefer making it similar to a curl command or replication:
```
/api/v1/upgrade

{ 
  source: "https://https://staging.dev.medicmobile.org/_couch/builds/docID",
  target: "/_couch/medic/_design/medic"
}
```
source and target are both just URLs or paths.